### ISSUE TITLE
view-source should return entire definition for custom command

### ISSUE THREAD

**Issue by NotTheDr01ds** [nushell/nushell#5373]
### Related problem

Define a custom command:

```
def foo [ bar:string ] {
    echo $"Hello, ($bar)"
}
```

View the definition of `foo`:

```
> view-source foo
{
    echo $"Hello, ($bar)"
}
```

The parameter list is lost.

### Describe the solution you'd like

Have `view-source` return the entire custom command definition, suitable for clipboard copy/paste and enough info to completely reproduce the definition.

```
> view-source foo
def foo [ bar:string ] {
    echo $"Hello, ($bar)"
}
```

Other uses:
```
> view-source foo | clip.exe # Windows 
> view-source foo | xclip -i -sel clipboard # Linux/Unix
> view-source foo > ~/myscript.nu
```

**Comment by kubouch** [comment]
Yeah, it would be nice. `view-source` is just a quick hack and if I recall correctly, there wasn't an easy way to fetch the full signature.

**Comment by onthebridgetonowhere** [comment]
Fixed in #5435.
